docs/src/how-to-use.md

```diff
@@ -337,3 +337,21 @@ Depending on the configuration of the energy storage assets, it may or may not b
 If no value is set in the parameter `use_binary_storage_method`, i.e., `missing` value, the storage asset can charge and discharge simultaneously.
 
 For more details on the constraints that apply when selecting this method, please visit the [`mathematical formulation`](@ref formulation) section.
+
+## [Setup a maximum or minimum outgoing energy limit](@id max-min-outgoing-energy-setup)
+
+To enable the model to utilize the constraints allowing us to consider a [maximum or minimum energy limit](@ref inter-temporal-energy-constraints) for an asset throughout the model's timeframe (e.g., a year), we need to establish a couple of parameters:
```

Comment:
Wow that's a meaty lead-in phrase.
Here's my attempt at shortening it:
```suggestion
For the model to add constraints for a [maximum or minimum energy limit](@ref inter-temporal-energy-constraints) for an asset throughout the model's timeframe (e.g., a year), we need to establish a couple of parameters:
```

If that's correct.